Tests/DatadogTests/Datadog/Mocks/RUMFeatureMocks.swift

```diff
@@ -105,10 +137,8 @@ class RUMScopeMock: RUMScope {
         activeUserActionID: nil
     )
 
-    var recordedCommands: [RUMCommand] = []
-
     func process(command: RUMCommand) -> Bool {
-        recordedCommands.append(command)
+        queue.async { self.commands.append(command) }
```

Comment:
i think we should do `expectation.fulfill()` here as well

Reply:
Yes! Added 👌.